paper.tex

```diff
@@ -401,7 +401,7 @@ \subsubsection{End-to-End-Encryption}
 The E2EE protocol presented in this paper combines a series of different cryptographic procedures. Each of them can be chosen on the basis of the working hardware and scenario by developers. On some certain devices, hardware acceleration would be supported. With hardware acceleration, the elapsed time could be dropped a step more. For example applying AES-NI can accelerate AES operations around 40\% on Intel devices.\cite{AES-NI-Acceleration} Fig.~\ref{fig:AES-NI} shows the performance of Encryption/Decryption of AES256 with and without hardware acceleration.
 
 \subsubsection{Register with Tangle-accelerator}
-User with channel seed can acquire channel chain ownership. Seed is generated by unique hardware information on edge device, so it should be transmitted to tangle-accelerator which allows it to publish messages on MAM channel chain. Asymmetric encryption is utilizing during the process of exchanging seed. Digital signature should be taken at the same time to ensure data integrity. Once the seed has been successfully exchanged, edge device is successfully registered on the specific tangle-accelerator. Thus, seed will not be exposed during communication after an edge device is successfully registered.
+User with channel seed can acquire channel chain ownership. Seed is generated by unique hardware information on edge device, so it should be transmitted to tangle-accelerator which allows it to publish messages on MAM channel chain. Once tangle-accelerator receive the Seed from edge device, then tangle-accelerator would return a user-key which can be mapping to corresponding Seed when edge devices send requests. Asymmetric encryption is utilizing during the process of exchanging seed. Digital signature should be taken at the same time to ensure data integrity. Once the seed has been successfully exchanged, edge device is successfully registered on the specific tangle-accelerator. Thus, seed will not be exposed during communication after an edge device is successfully registered.
```

Comment:
* user-key -> unique identifier
* Once Tangle-accelerator receive....., it would....

Reply:
How about using `unique user identifier` to replace `user key`?

Reply:
OK to rework?